README.md

```diff
@@ -6,16 +6,16 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
 Cellery is a code-first approach to building, integrating, running and managing composite microservice applications on Kubernetes. 
-Build, push/pull, run, scale and observe composites. Create secure APIs by default, code components in any language.
+Build, push/pull, run, scale and observe composites. Secure APIs by default. Code in any language.
```

Comment:
@isurulucky , some of these changes are performed by Paul, I think you have mistakenly reverted those.

Reply:
Fixed with latest commit to this PR.